shared/profile/container.js

```diff
@@ -68,9 +41,8 @@ class ProfileContainer extends PureComponent<void, EitherProps<Props>, State> {
 
     return <Profile
       {...props}
-      onAvatarLoaded={this._onAvatarLoaded}
-      followers={this.state.avatarLoaded ? props.followers : null}
-      following={this.state.avatarLoaded ? props.following : null} />
+      followers={props.followers}
```

Comment:
Does this change the behavior to now show followers/following before the avatar loads?